packages/@react-spectrum/searchwithin/test/SearchWithin.test.js

```diff
@@ -152,20 +152,40 @@ describe('SearchWithin', function () {
 
     expect(searchfield).not.toHaveAttribute('aria-required');
 
-    expect(searchfield).toHaveAttribute('aria-labelledby', label.id);
-    expect(group).toHaveAttribute('aria-labelledby', label.id);
+    expect(searchfield).toHaveAttribute('aria-labelledby', `${label.id} ${group.id}`);
+    expect(group).toHaveAttribute('aria-labelledby', `${label.id} ${group.id}`);
 
     expect(searchfield.classList.contains('is-quiet')).toBeFalsy();
     expect(picker.classList.contains('spectrum-Dropdown--quiet')).toBeFalsy();
   });
 
-  it('searchfield and group are labelled correctly if aria-label used', function () {
-    let {getByRole, getByLabelText} = renderSearchWithin({label: undefined, 'aria-label': 'Aria Label'});
+  it('Should handle aria-labels without visible label', function () {
+    let {getByRole} = renderSearchWithin({label: undefined, 'aria-label': 'Aria Label'});
 
-    let searchfield = getByLabelText('Aria Label', {selector: 'input'});
     let group = getByRole('group');
+    let searchfield = getByRole('searchbox');
+    let picker = getByRole('button');
+
+    expect(group).toHaveAttribute('aria-label', 'Aria Label');
+    expect(picker).toHaveAttribute('aria-label', 'Search within');
+
+    expect(group).not.toHaveAttribute('aria-labelledby');
+    expect(searchfield).toHaveAttribute('aria-labelledby', group.id);
+    expect(picker).toHaveAttribute('aria-labelledby', `${group.id} ${picker.id} ${picker.childNodes[0].id}`);
+  });
+
+  it('Should use default aria-labels if no aria-label provided', function () {
+    let {getByRole} = renderSearchWithin({label: undefined});
+
+    let group = getByRole('group');
+    let searchfield = getByRole('searchbox');
+    let picker = getByRole('button');
+
+    expect(group).toHaveAttribute('aria-label', 'Search');
+    expect(picker).toHaveAttribute('aria-label', 'Search within');
 
-    expect(searchfield).toHaveAttribute('aria-label', 'Aria Label');
-    expect(group).toHaveAttribute('aria-labelledby', 'Aria Label');
+    expect(group).not.toHaveAttribute('aria-labelledby');
+    expect(searchfield).toHaveAttribute('aria-labelledby');
```

Comment:
Why don't these have `group.id`?

Reply:
Think you are right, maybe it should be:
```suggestion
    expect(group).not.toHaveAttribute('aria-labelledby');
    expect(searchfield).toHaveAttribute('aria-labelledby',  group.id);
```
cc @reidbarber

Reply:
Good catch, pushing a fix.